subprojects/docs/src/docs/stylesheets/userGuideHtml.xsl

```diff
@@ -171,10 +171,25 @@
                                     <li><a href="/userguide/feature_lifecycle.html">Feature Lifecycle</a></li></ul></li>
                             <li><a class="nav-dropdown" data-toggle="collapse" href="#managing-dependencies" aria-expanded="false" aria-controls="managing-dependencies">Managing Dependencies</a>
                                 <ul id="managing-dependencies">
-                                    <li><a href="/userguide/artifact_dependencies_tutorial.html">Dependency Management Basics</a></li>
-                                    <li><a href="/userguide/dependency_management.html">Advanced Dependency Management</a></li>
-                                    <li><a href="/userguide/dependency_types.html">Dependency Types</a></li>
-                                    <li><a href="/userguide/repository_types.html">Repository Types</a></li></ul></li>
+                                    <li><a class="nav-dropdown" data-toggle="collapse" href="#dependency-management-tutorials" aria-expanded="false" aria-controls="dependency-management-tutorials">Tutorials</a>
+                                        <ul id="dependency-management-tutorials">
+                                            <li><a href="/userguide/artifact_dependencies_tutorial.html">Dependency Management for Java Projects</a></li>
+                                        </ul>
+                                    </li>
+                                    <li><a href="/userguide/dependency_management_introduction.html">Introduction</a></li>
+                                    <li><a href="/userguide/declaring_dependencies.html">Declaring Dependencies</a></li>
+                                    <li><a href="/userguide/declaring_repositories.html">Declaring Repositories</a></li>
+                                    <li><a href="/userguide/inspecting_dependencies.html">Inspecting Dependencies</a></li>
+                                    <li><a href="/userguide/managing_transitive_dependencies.html">Managing Transitive Dependencies</a></li>
+                                    <li><a href="/userguide/working_with_dependencies.html">Working with dependencies</a></li>
+                                    <li><a href="/userguide/customizing_dependency_resolution_behavior.html">Customizing Dependency Resolution Behavior</a></li>
+                                    <li><a href="/userguide/troubleshooting_dependency_resolution.html">Troubleshooting Dependency Resolution</a></li>
+                                    <li><a class="nav-dropdown" data-toggle="collapse" href="#dependency-management-glossary" aria-expanded="false" aria-controls="dependency-management-glossary">Glossary</a>
+                                        <ul id="dependency-management-glossary">
+                                            <li><a href="/userguide/dependency_types.html">Dependency Types</a></li>
```

Comment:
One really glaring omission in this whole thing is "Configurations". I would consider a single "Concepts" page instead of splitting this into multiple pages. This could be embedded in the introduction.

Reply:
We actually already have coverage on "configurations". It's just not labeled that way. https://docs.gradle.org/4.5-rc-1/userguide/dependency_management.html#sub:scope_of_dependency_configurations

In a way a "configuration" is a concept but only really makes sense in the context of declaring a dependency. I guess we have the following options:

1. Leave the content where it is with no change. The introduction will explain what a "configuration" is and link to the existing section.
2. Move the content into a completely new page similar to "Dependency types" and "Repository types".
2. Leave the content where it is and change the title to something with "configuration".